src/pb_env.cc

```diff
@@ -253,6 +253,18 @@ EnvironmentManager::ExtractIfNotExtracted(std::string env_path)
 
   bool env_extracted = false;
   bool re_extraction = false;
+
+  // If the path is not a conda-packed file, then bypass the extraction process
+  std::string subPath = env_path.substr(env_path.size() - 6);
```

Comment:
I think this can throw an error. If the env_path is < 6 characters, size_t will interpret it as a very large number (close to the max possible value), which will throw an out of range.

Reference: https://cplusplus.com/reference/string/string/substr/